### PR TITLE
Force no oechem

### PR DESCRIPTION
**Pull request by richardjgowers**

### CODE REVIEW

**Comment by richardjgowers** [comment]
@IAlibay this is a pain to test inside the protocol but I've got some eg5 edges working with oechem present with this patch.

**Comment by IAlibay** [comment]
I mean this is fine, but I don't understand why we needed to vendor this?

File: openfe/protocols/openmm_rfe/equil_rfe_methods.py [hunk 1]
```diff
@@ -679,7 +680,9 @@ def run(self, *, dry=False, verbose=True,
     def _execute(
         self, ctx: gufe.Context, **kwargs,
     ) -> dict[str, Any]:
-        outputs = self.run(scratch_basepath=ctx.scratch, shared_basepath=ctx.shared)
+        with without_oechem_backend():
```
Comment (IAlibay) [inline openfe/protocols/openmm_rfe/equil_rfe_methods.py:683]: Actually we might need to review this, I suspect we'll run into trouble when we start offering folks the ability to choose what charge derivation method they can have (for example if they want elf charges)

**Review by IAlibay — APPROVED** [review]
Let's go with this for now, but we'll need to make it optional eventually (probably when it's fixed in openmmforcefields) - that way folks can choose different flavours of charge eventually.

**Pull request merged**